scripts/apps/authoring/views/full-preview-item.html

```diff
@@ -147,7 +147,7 @@
         </div>
 
         <div ng-if="!hideMedia && item.type === 'picture' && item.renditions.original.href" style="margin-bottom: 20px">
```

Comment:
would be good to change it here as well

Reply:
Here's fix PR https://github.com/superdesk/superdesk-client-core/pull/3627